stock_inventory_preparation_filter/README.rst

```diff
@@ -15,6 +15,10 @@ Includes more options for making an inventory out of:
 It also allows to make an inventory based on scanned products, adding a line
 with product code and quantity.
 
+Also, if "All Products" or "Selected Categories" options chosen, then you can select between importing:
```

Comment:
I think I better delete the code modification over lots because it doesn't make any sense there, when choosing lots you are choosing products or not? 
What do yo think?